api/lib/PkgFsBindings.ts

```diff
@@ -0,0 +1,86 @@
+import { FileHandle, FSStats, type FileSystem } from '@zwave-js/shared/bindings'
+import { fs as nodeFs } from '@zwave-js/core/bindings/fs/node'
+import path from 'node:path'
+
+// Ensures that the Z-Wave JS driver is looking for the right files in the right place
+// when running inside a `pkg` bundle. In this case, it will resolve its embedded
+// configuration dir to the path "/config", but the files reside in "node_modules/@zwave-js/config/config" instead.
+
+const CONFIG_PATH = '/config'
+const CONFIG_PATH_IN_PKG = path.join(
+	__dirname,
+	`node_modules/@zwave-js/config/config`,
+)
```

Comment:
I'm wondering if this could be fixed in esbuild bundle by simply monkey patching the file and replacing the place where you set this path?

Reply:
I'm afraid that will just lead to us chasing our tail again when I find the need to change the specific code again. Right now it's relatively specific to handle 3 cases: file executed as JS, file executed as TS, run in a context where there is no "main" file:
```ts
/** The absolute path of the embedded configuration directory */
export const configDir = import.meta.url.startsWith("file:")
	? path.join(
		path.dirname(fileURLToPath(import.meta.url)),
		import.meta.url.endsWith("src/utils.ts")
			? ".."
			: "../..",
		"config",
	)
	: import.meta.resolve("/config");
```